cfme/tests/configure/test_register_appliance.py

```diff
@@ -181,6 +182,9 @@ def test_rh_registration(
     Bugzilla:
         1532201
     """
+    bug = GH('ManageIQ/manageiq:20390')
```

Comment:
Please use the existing `meta` marker and include the `blockers` kwarg, as it supports GH issues.

Reply:
@mshriver Thanks. I planned to make the skip conditional and forgot about that. However, with the current configuration it blocks all the tests anyway.